code/modules/surgery/organs_internal.dm

```diff
@@ -156,6 +156,14 @@
 					"\blue You treat damage to [target]'s [I.name] with [tool_name]." )
 					I.damage = 0
 
+		if(target_zone == "head")
```

Comment:
Organs should probably just have an on_repair() proc or something that this surgery would then call.